net/dispatcher_test.go

```diff
@@ -146,7 +146,7 @@ func TestDispatcher_SendRequest(t *testing.T) {
 				defer deferFn()
 			}
 
-			got, err := d.SendRequest(tt.args.endpoint, tt.args.method, tt.args.jsonData, tt.args.signatureHeader, tt.args.hmac)
+			got, err := d.SendRequest(tt.args.endpoint, tt.args.method, tt.args.jsonData, tt.args.signatureHeader, tt.args.hmac, config.MaxResponseSize)
```

Comment:
@danvixent We should have a test case to catch reading more bytes than configured.

Reply:
done